doc/project_governance/technical_committee.md

```diff
@@ -27,3 +31,85 @@ Other individuals are directly appointed by the OpenTitan Governing Board.
 - Shimeon Greenberg (Nuvoton)
 - Rupert Swarbrick (lowRISC)
 - No current appointee (Giesecke & Devrient)*
+
+## Roles {#roles}
+
+The OTTC Chair facilitates the effective functioning of the OTTC.  
+The role confers no enhanced voting rights.  
+By default the OTTC Chair is the lowRISC representative in the OTTC.  
+Any alternative OTTC member may be appointed to the Chair for a period of three months by OTTC vote.
+
+## Appointing members {#appointing-members}
+
+Membership of the OTTC is defined in the OpenTitan [Technical Charter](https://static.opentitan.org/technical-charter.pdf).
+
+## Meetings {#meetings}
+
+OTTC meetings will be held regularly on an agreed schedule. 
+The Chair may call additional meetings when a discussion or decision is needed outside the schedule.  
+Minutes should be kept in a form accessible to OTTC members but will be considered confidential to this group. 
+If a meeting has no agenda items (see below), the Chair may cancel it ahead of time.
+
+### Notification of meetings {#notification-of-meetings}
+
+Agenda items should be submitted at least three days in advance of the meeting by adding them to the meeting notes doc.  
+The Chair should circulate the agenda at least two days in advance of the meeting.
+
+### Voting and quoracy {#voting-and-quoracy}
+
+Voting uses the following rules unless otherwise specified.
+
+* Voting may be at a meeting to which all TC members are invited or offline by email on the TC mailing list.    
+* If voting is offline, at least two weeks’ notice is required.  
+* If a TC member cannot attend a meeting to vote, they may instead vote by email to the TC mailing list before the respective TC meeting.  
+* For a vote to be quorate, it requires:  
+  * At least 60% of OTTC members to respond “Yes” or “No”  
+* For a vote to pass, it requires  
+  * At least 75% of those OTTC members who vote “Yes” or “No” to be in favour
+
+The chair has no enhanced voting rights as a result of their position.  
+Voting results are recorded in the TC meeting notes. Results may be captured by number (e.g., “5 Yes, 2 No”) or by name (e.g., “Yes: Member A, Member C; No: Member B”).
+
+### Communication {#communication}
+
+Decisions and action points should be recorded and distributed to OTTC members and the OTGB no later than a week after the meeting.  
+Where OTTC decisions affect the broader OpenTitan community, the Chair should ensure that information is sent, or documentation updated, in a timely manner.
+
+## Responsibilities {#responsibilities}
+
+### Maintaining the list of OpenTitan Committers. {#maintaining-the-list-of-opentitan-committers.}
+
+Committers are individuals with repository write access. 
+While everyone can and is encouraged to contribute by reviewing code, committers are responsible for final approval and merge.  
+New committers are appointed by OTTC vote.  
+The OTTC removes committers who no longer meet the OTTC’s criteria.
```

Comment:
Which criteria?

Reply:
The TC has agreed policies for committer approval and committer removal. Please talk to your TC rep if you think a contributor should become a committer or no longer be a committer.

Reply:
My point is if you refer to a criteria, link it here. Otherwise the document becomes vague.